### PR TITLE
Simulator: remove carla

### PR DESCRIPTION
**Pull request by jnewb1**
carla has been replaced by metadrive in openpilot as it is much simpler and runs better

https://blog.comma.ai/095release/#metadrive-our-new-default-simulator-experience

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ All openpilot services can run as usual on a PC without requiring special hardwa
 
 With openpilot's tools, you can plot logs, replay drives, and watch the full-res camera streams. See [the tools README](tools/README.md) for more information.
 
-You can also run openpilot in simulation [with the CARLA simulator](tools/sim/README.md). This allows openpilot to drive around a virtual car on your Ubuntu machine. The whole setup should only take a few minutes but does require a decent GPU.
+You can also run openpilot in simulation [with the metadrive simulator](tools/sim/README.md). This allows openpilot to drive around a virtual car on your Ubuntu machine. The whole setup should only take a few minutes but does require a decent GPU.
```
Comment (adeebshihadeh) [inline README.md:61]: it's stylized as MetaDrive

File: tools/sim/run_bridge.py [hunk 2]
```diff
@@ -17,22 +16,14 @@ def parse_args(add_args=None):
...
 
 if __name__ == "__main__":
   q: Any = Queue()
   args = parse_args()
 
   simulator_bridge: SimulatorBridge
-  if args.simulator == "carla":
-    simulator_bridge = CarlaBridge(args)
-  elif args.simulator == "metadrive":
+  if args.simulator == "metadrive":
```
Comment (adeebshihadeh) [inline tools/sim/run_bridge.py:26]: let's remove the arg

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]

File: tools/sim/run_bridge.py (hunk 2 above)
Comment (jnewb1) [reply tools/sim/run_bridge.py]: done

File: README.md (hunk 1 above)
Comment (jnewb1) [reply README.md]: done

**Comment by mimi89999** [comment]
Maybe the wiki should be updated too https://github.com/commaai/openpilot/wiki/Simulation ?

**Pull request merged**